go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
 	github.com/ethereum-optimism/go-ethereum-hdwallet v0.1.3
-	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20231001123245-7b48d3818686
-	github.com/ethereum/go-ethereum v1.13.1
+	github.com/ethereum-optimism/superchain-registry/superchain v0.0.0-20231018164214-046f42968aec
```

Comment:
Also good to see this updated to match op-geth update :+1:

Reply:
Interestingly, that wasn't me. go mod tidy did it automatically which is a bonus.